build.gradle.kts

```diff
@@ -1,7 +1,6 @@
 plugins {
     alias(libs.plugins.kotlin.jvm) apply false
     alias(libs.plugins.kotlin.kapt) apply false
-    alias(libs.plugins.specificlanguages.mps) apply false
```

Comment:
Seems not to be used 🤷

Reply:
That would be great as a commit and PR description.